src/isak_finnoy_ex/ex02/message_entropy.py

```diff
@@ -0,0 +1,20 @@
+__author__ = 'Isak Finnoy'
+__email__ = 'isfi@nmbu.no'
+
+
+def entropy(message):  # estimates the entropy of a string
+    import math   # imports the math module
+    import letter_counts  # imports self-made module
+    freq = letter_counts.letter_freq(message)  # uses letter_freq to count frequency of characters in string
+    teller = len(message)  # sets the denominator equal to the length of the string
+    liste = []   # creates an empty list
+    for i in freq.keys():  # loops over the keys in the dictionary letter_freq returns
```

Comment:
`i` is not used as individual role, maybe you want to change `freq.keys()` to `freq.values()`